vault/login_mfa.go

```diff
@@ -1402,20 +1383,19 @@ func (c *Core) validateLoginMFAInternal(ctx context.Context, methodID string, en
 		return fmt.Errorf("MFA method configuration not present")
 	}
 
-	var alias *identity.Alias
 	var finalUsername string
 	switch mConfig.Type {
 	case mfaMethodTypeDuo, mfaMethodTypeOkta, mfaMethodTypePingID:
```

Comment:
Think we want something more like https://github.com/hashicorp/vault/blob/main/vault/policy.go#L293